decidim-assemblies/lib/decidim/api/assembly_type.rb

```diff
@@ -6,17 +6,20 @@ module Assemblies
     class AssemblyType < Decidim::Api::Types::BaseObject
       implements Decidim::Core::ParticipatorySpaceInterface
       implements Decidim::Core::AttachableInterface
+      implements Decidim::Core::AttachableCollectionInterface
       implements Decidim::Core::ParticipatorySpaceResourceableInterface
       implements Decidim::Core::TaxonomizableInterface
       implements Decidim::Core::CategoriesContainerInterface
```

Comment:
I think we should remove Categories from here (and the specs)

Reply:
TBH... This needs to be done in a standalone PR (we still have the following types : area, area_type, category, scope ) and also interfaces: (CategoriesContainerInterface, CategorizableInterface, ScopableInterface).

Reply:
ah perfect!